src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -0,0 +1,35 @@
+import { ThemeOptions } from '@mui/material/styles'
+import { typographyOptions } from './typography/Typography'
+import palette from './palette/Palette'
+import { Fade } from '@mui/material'
+
+const defaultMuiTheme: ThemeOptions = {
+  typography: typographyOptions,
+  palette: palette,
+  components: {
+    MuiTooltip: {
+      defaultProps: {
+        arrow: true,
+        TransitionComponent: Fade,
+      },
+      styleOverrides: {
+        arrow: ({ ownerState, theme }) => ({
+          color: theme.palette.common.black,
+        }),
+        tooltip: ({ ownerState, theme }) => ({
+          fontSize: '14px',
+          borderRadius: '2px',
+          backgroundColor: theme.palette.common.black,
+        }),
+      },
+    },
```

Comment:
Instead of applying in a wrapper component over tooltip, we set these in the theme and then they are always applied to the Tooltip component. Wrapper component removed.